cscs-checks/apps/quantumespresso/quantumespresso_check.py

```diff
@@ -0,0 +1,139 @@
+# Copyright 2016-2020 Swiss National Supercomputing Centre (CSCS/ETH Zurich)
+# ReFrame Project Developers. See the top-level LICENSE file for details.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+import os
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+
+class QuantumESPRESSOCheck(rfm.RunOnlyRegressionTest):
+    def __init__(self):
+        self.valid_prog_environs = ['builtin']
+        self.executable = 'pw.x'
+        self.executable_opts = ['-in', 'ausurf.in']
+
+        self.sanity_patterns = sn.all([
+            sn.assert_found(r'convergence has been achieved', self.stdout),
+        ])
+
+        self.perf_patterns = {
+            'time': sn.extractsingle(r'electrons.+\s(?P<wtime>\S+)s WALL',
+                                     self.stdout, 'wtime', float)
+        }
+
+        self.maintainers = ['LM']
+        self.tags = {'scs'}
+        self.strict_check = False
+        self.extra_resources = {
+            'switches': {
+                'num_switches': 1
+            }
+        }
+
+
+@rfm.parameterized_test(*([s, v]
+                          for s in ['small', 'large']
+                          for v in ['maint', 'prod']))
+class QuantumESPRESSOCpuCheck(QuantumESPRESSOCheck):
+    def __init__(self, scale, variant):
+        super().__init__()
+        self.descr = 'QuantumESPRESSO CPU check (version: %s, %s)' \
+                     % (scale, variant)
```

Comment:
Rewrite this using f-strings:

```suggestion
        self.descr = f'QuantumESPRESSO CPU check (version: {scale}, {variant})'
```